content/en/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md

```diff
@@ -3,27 +3,26 @@ title: Extend the Kubernetes API with CustomResourceDefinitions
 reviewers:
 - deads2k
 - enisoc
-content_template: templates/task
 ---
 
-{{% capture overview %}}
```

Comment:
These captures and admonitions need to be reverted back to the versions in the red lines, because they were already fixed in `release-1.11` branch but your PR (inadvertently) changes them back. Please remove the changes from this PR. I apologize for the churn.